.github/workflows/check-licenses-npm-package.yaml

```diff
@@ -88,5 +88,5 @@ jobs:
           cat package.json
           echo "========"
           npm i
-          npm install license-checker-rseidelsohn --save-dev
-          npx license-checker-rseidelsohn --summary --excludePrivatePackages --onlyAllow 'MIT;ISC;Apache-2.0;BSD-2-Clause;BSD-3-Clause;BSD-4-Clause;CC-BY-3.0;CC-BY-4.0;BlueOak-1.0.0;CC0-1.0;0BSD;Python-2.0;BSD*;Unlicense'
```

Comment:
is this fix for other Repository? or only for the juno and what is exactly fixing?

Reply:
for greenhouse and greenhouse-extensions